server/routes/api/users.js

```diff
@@ -75,4 +76,78 @@ router.post("/login", (req, res) => {
   }
 });
 
+// @route GET /api/users/{id}
+// @desc Get a User object
+// @access Authenticated User can get own record
+router.get("/:id", (req, res) => {
+  let id = req.params.id;
+  User.findById(id, function(err, user) {
+    if (!user) {
+      res.status(404).send({ id: `User with id ${id} is not found` });
+    } else {
+      res.json(user);
```

Comment:
I believe this will also send the password hash to the front-end, don't want to do that - you can probably make a custom json representation of the user

Reply:
Good point! Fixed.